2.0/m_geoipban.cpp

```diff
@@ -13,7 +13,7 @@
  * details.
  *
  * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * along with this program.	 If not, see <http://www.gnu.org/licenses/>.
```

Comment:
This doesn't need to change?

Reply:
Ops ... something goes wrong using coda2 plugin converting identation ... this night i'll make a new commit.